python_modules/libraries/dagster-dg/dagster_dg/component.py

```diff
@@ -22,22 +21,46 @@ class RemoteComponentType:
     component_schema: Optional[Mapping[str, Any]]  # json schema
 
 
-def _get_remote_type_mapping_from_raw_data(
-    raw_data: Mapping[str, Any],
-) -> Mapping[ComponentKey, RemoteComponentType]:
-    data = {}
-    for typename, metadata in raw_data.items():
-        data[GlobalComponentKey.from_typename(typename)] = RemoteComponentType(**metadata)
-    return data
+class RemoteComponentRegistry:
```

Comment:
We can probably reconsider the naming of this eh?

Reply:
Maybe-- it's true that there's no longer a corresponding plain `ComponentRegistry`, but I think `RemoteComponentRegistry` is still a sensible name.

Reply:
anyways does not need to block this PR. It's an internal name so its a two way door